### PR TITLE
Configuration of the B2B sample application is refactored

### PR DESCRIPTION
**Pull request by Achintha444**
```
{
    "AuthorizationConfig": {
        "BaseOrganizationUrl": "<PARENT ORGANIZATION URL> ex: https://dev.api.asgardeo.io/t/guardio",
        "ClientId": "<CLIENT ID>",
        "ClientSecret": "<CLIENT SECRET>"
    },
    "ApplicationConfig": {
        "HostedUrl": "http://localhost:3000",
        "APIScopes": [
            "openid",
            "email",
            "profile",
            "internal_login",
            "internal_user_mgt_view",
            "internal_user_mgt_list",
            "internal_user_mgt_update",
            "internal_user_mgt_delete",
            "internal_user_mgt_create",
            "internal_idp_view",
            "internal_idp_create",
            "internal_idp_update",
            "internal_idp_delete",
            "internal_application_mgt_view",
            "internal_application_mgt_update",
            "internal_organization_view",
            "internal_role_mgt_view",
            "internal_role_mgt_create",
            "internal_role_mgt_update",
            "internal_role_mgt_delete",
            "internal_group_mgt_update"
        ],
        "Branding": {
            "name": "Guardio Insurance",
            "tag": "Anytime . Anywhere"
        },
        "SampleOrganization" : [
            {
                "id": "<SUB ORGANIZATION ID>",
                "name": "<SUB ORGANIZATION NAME>"
            }
        ]
    },
    "ManagementAPIConfig": {
        "SharedApplicationName": "<SHARED APPLICATION NAME> ex: guardio-app",
        "ImageBaseUrl" : "<IMAGE BASE URL - FOLLOW README.MD FOR MORE INFORMATION>"
    }
}
```

### CODE REVIEW

File: b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js [hunk 1]
```diff
@@ -16,29 +16,37 @@
  * under the License.
  */
 
-import config from "../../../config.json";
 import getDataHeader from "../../../util/util/apiUtil/getDataHeader";
 import { dataNotRecievedError, notPostError } from "../../../util/util/apiUtil/localResErrors";
+import { getOrgUrl } from "../../../util/util/orgUtil/orgUtil";
 
+/**
+ * backend API call to get logged user details
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js:24]: Let's start the comment with an upper-case letter and ends with a full-stop. Check other places too.

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/confirmAddRemoveLoginFlowModal.js [hunk 2]
```diff
@@ -126,7 +126,8 @@ export default function ConfirmAddRemoveLoginFlowModal(prop) {
 
 /**
  * 
- * @returns When then WSO2IS_APP_NAME is not the correct applicaiton, it will show this section
+ * @returns When then `config.ManagementAPIConfig.SharedApplicationName` is not the correct applicaiton, 
+ * it will show this section
```
Comment (ThaminduR) [inline b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/confirmAddRemoveLoginFlowModal.js:130]: Shall we have a `.` at the end. Check other related places as well.

File: b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getUrls.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * URL extracted from the `config.AuthorizationConfig.BaseOrganizationUrl`
+ * 
+ * @returns get managemnt API server base URL
+ */
+function getManagementAPIServerBaseUrl() {
+
+    // todo: implementation will change after changes are done to the IS.
+
```
Comment (ThaminduR) [inline b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getUrls.js:29]: let's remove the extra line. Seems it's not needed.

File: b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js (hunk 1 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js:24]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/confirmAddRemoveLoginFlowModal.js (hunk 2 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/confirmAddRemoveLoginFlowModal.js:130]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b

File: b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getUrls.js (hunk 3 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getUrls.js:29]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b

**Pull request merged**